controllers/components/kserve/kserve_controller_actions.go

```diff
@@ -262,3 +263,23 @@ func customizeKserveConfigMap(ctx context.Context, rr *odhtypes.ReconciliationRe
 
 	return nil
 }
+
+func setStatusFields(ctx context.Context, rr *odhtypes.ReconciliationRequest) error {
+	k, ok := rr.Instance.(*componentApi.Kserve)
+	if !ok {
+		return fmt.Errorf("resource instance %v is not a componentApi.Kserve)", rr.Instance)
+	}
+
+	kserveConfigMap := corev1.ConfigMap{}
+	if err := rr.Client.Get(ctx, client.ObjectKey{Name: kserveConfigMapName, Namespace: rr.DSCI.Spec.ApplicationsNamespace}, &kserveConfigMap); err != nil {
+		return err
+	}
```

Comment:
these can be part of getDefaultDeploymentMode() call instead ?

Reply:
Yeah it probably can, I'll change that. If in the future we decide that we want to set status fields based on other things in the configmap, we can pull it out again.

Reply:
Done